src/scroll-timeline-css.js

```diff
@@ -5,7 +5,7 @@ import { ScrollTimeline, ViewTimeline, getScrollParent, calculateRange,
 
```

Comment:
Until we can get rid of the ScrollTimeline, ViewTimeline and ProxyAnimation imports I think the css polyfill will only work if the js polyfill was loaded. E.g. we use whether the returned animation is a ProxyAnimation to determine whether we need to set up a css scroll driven animation: https://github.com/flackr/scroll-timeline/blob/master/src/scroll-timeline-css.js#L157

We're similarly dependent on getAnimations being overridden to return the ProxyAnimation instances.